playbooks/bootstrap.yml

```diff
@@ -1,6 +1,35 @@
 ---
-- import_playbook: "./bootstrap_tower.yml"
-- import_playbook: "./bootstrap_integreatly.yml"
-- import_playbook: "./bootstrap_cluster_create.yml"
-- import_playbook: "./bootstrap_cluster_teardown.yml"
-- import_playbook: "./bootstrap_validation.yml"
\ No newline at end of file
+- hosts: localhost
+  gather_facts: no
+  tasks:
+    - debug:
+        msg: "ERROR: No branch/tag of the configuration repo defined defaulting to master"
+      when: integreatly_version is not defined
+
+    - set_fact:
+        tower_configuration_project_scm_branch: "master"
```

Comment:
We can probably just update the variable value directly in the tower role defaults/main.yml where it defaults to master. See example from installation repo: https://github.com/integr8ly/installation/blob/master/inventories/group_vars/all/common.yml#L104